docs/context/evidence/issue_3558_gate_threshold_sweep_2026-06-25/README.md

```
@@ -0,0 +1,58 @@
+  is genuinely contested, so the signal is the *relative* contrast (dropping raises collisions from
+  0.417 → 0.917), not an absolute safety claim.
+
+## Reproduce
+
+```bash
+uv run python scripts/validation/run_stream_gap_gate_threshold_sweep_issue_3558.py \
+  --output-json output/issue_3558_gate_threshold_sweep.json
+```
+
+Tests: `tests/validation/test_run_stream_gap_gate_threshold_sweep_issue_3558.py` (and the unchanged
+#3471 harness tests, which still pass with the additive `gate_thresholds` parameter).
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)